spec/marqeta/card_spec.rb

```diff
@@ -111,5 +111,12 @@ def fetch_card_from_pan
         expect(card.pan).to eq(pan)
       end
     end
+
+    describe '#create_client_access' do
+      it "creates a ClientAccess resource passing in the card's token" do
+        expect(Marqeta::ClientAccess).to receive(:api_create).with(card_token: card_token)
+        card.create_client_access
```

Comment:
@sughandj  Will there be a situation where the client access is not created?

Reply:
@teresa-lo ClientAccess is like OneTime, they last 5 mins. It's created on some of cards of our cards (not all)